local-helm/README.md

```diff
@@ -38,6 +38,10 @@ Create a secret in kubernetes to access the private container registry, as per t
 
 You may wish to run the dashboard. Instructions are [here](../Docs/DevSetup/run-dashboard.md).
 
+### (Optional) Run Rancher
+
+You may wish to run Rancher rather than the dashboard. Rancher provide a step by step walkthrough to install, located [here](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/helm-rancher/)
+
```

Comment:
I think some extra (linked to) instructions would be useful, e.g. how to set up a project for buyingcatalogue, link to the setup for the metric setup.
Also worth pointing out it will be running then on rancher.localhost.
The nginx ingress setup may also conflict with the one we have in our instructions, so it may be best to amend our setup ingress instruction to match.

Reply:
I didn't have to modify the buying catalogue nginx to get it to work with rancher, rancher itself did everything behind nginx. I'll add some more detailed instructions on Rancher setup though

Reply:
More detailed instructions added